src/components/SearchPage/SearchPageComponent.js

```diff
@@ -67,22 +51,22 @@ export function SearchPageComponent() {
 
   return (
     <>
-      <BackButtonWrapper>
-        <Link to="notes">
-          <IconButton aria-label="back">
-            <ArrowBackIcon style={{ fill: colors.primaryColor }} />
-          </IconButton>
-        </Link>
-      </BackButtonWrapper>
-      <input
-        defaultValue={defaultValue}
-        onChange={e => debounced.callback(e.target.value, setQuery)}
-      />
+      <TopBar>
+        <TopBarLeftButtons>
+          <Link to="notes">
+            <IconButton aria-label="back">
+              <ArrowBackIcon style={{ fill: colors.textColor }} />
```

Comment:
Please use the [BackButton](https://github.com/DarthVanger/NeuralNotes/blob/master/src/components/BackButton/BackButton.js) component to avoid duplicating the same code and styles.